iotdb-core/ainode/ainode/core/manager/inference_manager.py

```diff
@@ -52,6 +52,17 @@ def infer(self, full_data, predict_length=96, **_):
         df = pd.DataFrame(output[0])
         return convert_to_binary(df)
 
+class SundialStrategy(InferenceStrategy):
+    def infer(self, full_data, predict_length=96, **_):
+        data = full_data[1][0]
+        if data.dtype.byteorder not in ('=', '|'):
+            data = data.byteswap().newbyteorder()
+        seqs = torch.tensor(data).unsqueeze(0).float()
```

Comment:
Why this? Is the input of Sundial and timerxl differenet?
By the way, I think we should not transfer `float` here. It should be in the period of data preproccessing.

Reply:
Yes, the current input are different, the demanding input class is tensor rather than nparray cuz we invoke the `generate` func for sundial. Let's leave the data processing logic here right now and unify this when we refactoring the integration of TimerXL